macros/commands/get_tables_without_model.sql

```diff
@@ -0,0 +1,59 @@
+{% macro get_tables_without_models(deprecated_models_path=none) %}
+  {% set model_schemas, model_tables = elementary.get_model_schemas_and_tables(deprecated_models_path) %}
+  {% set db_tables = elementary.get_db_tables_in_schemas(model_schemas) %}
+  {% for db_table in db_tables %}
+    {% if db_table not in model_tables %}
+      {% do print(db_table) %}
+    {% endif %}
+  {% endfor %}
+{% endmacro %}
+
+
+{% macro get_db_tables_in_schemas(schemas) %}
+  {% set tables = [] %}
+  {% for schema in schemas %}
+    {% set db_name, schema_name = schema.split('.') %}
+    {% set schema_relation = api.Relation.create(db_name, schema_name).without_identifier() %}
+    {% set relations = dbt.list_relations_without_caching(schema_relation) %}
+    {# list_relations_without_caching can return either a list of Relation objects or an agate depending on the adapter #}
+    {% if relations.append is defined %}
+      {# relations is a list of Relation objects #}
+      {% for relation in relations %}
+        {% do tables.append(schema ~ "." ~ relation.identifier) %}  
```

Comment:
I usually prefer using `api.Relation.create` as it takes care of stringifying the relation in the safest way (quoting, etc.).